benchmarks/benchmarks/stats.py

```diff
@@ -57,45 +57,107 @@ def time_ttest_ind_diff_var(self):
 
 
 class Distribution(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that are files under `distcont` and `distdiscrete`.
+    dists = sorted(list(set([d[0] for d in distcont + distdiscrete])))
+
     param_names = ['distribution', 'properties']
     params = [
-        ['cauchy', 'gamma', 'beta'],
-        ['pdf', 'cdf', 'rvs', 'fit']
+        dists, ['pdf/pmf', 'logpdf/logpmf', 'cdf', 'logcdf', 'rvs', 'fit',
+                'sf', 'logsf', 'ppf', 'isf', 'moment', 'stats_s', 'stats_v',
+                'stats_m', 'stats_k', 'stats_mvsk', 'entropy']
     ]
+    distcont = dict(distcont + distdiscrete)
```

Comment:
```suggestion
    dists = dict(distcont + distdiscrete)
```
"cont" stands for continuous. These are continuous and discrete.

Reply:
changed to dist_data